scipy/stats/_continuous_distns.py

```diff
@@ -8426,6 +8433,17 @@ def _stats(self, beta):
     def _entropy(self, beta):
         return 1. / beta - np.log(.5 * beta) + sc.gammaln(1. / beta)
 
+    def _rvs(self, beta, size=None, random_state=None):
+        # see [2]_ for the algorithm
+        # see [3]_ for reference implementation in SAS
+        z = random_state.gamma(1/beta, size=size)
```

Comment:
I'm forgetting how broadcasting is supposed to work. The `size` the user enters is always exactly the output shape of the array, right? The array `beta` that the user enters needs to broadcast with an array of that shape, right?

This is in contrast to, say, the user entering beta = [1, 2] and the size being 1000 and getting a 2D array out, right?

Reply:
> The `size` the user enters is always exactly the output shape of the array, right? The array `beta` that the user enters needs to broadcast with an array of that shape, right?

Yes and yes.

> This is in contrast to, say, the user entering `beta=[1, 2]` and the `size=1000` and getting a 2D array out, right?

Yes. The user should pass `size=[1000, 2]` instead otherwise `rvs` would just raise an error.

Reply:
OK. I just forget these things. Note to self: explain this in the definition of `size` for distribution rvs methods.

![image](https://user-images.githubusercontent.com/6570539/149612296-4409ebd1-a80c-4f83-adde-20aa3bc33b18.png)

Maybe
> The shape of the output array of random variates. By default, `rvs` returns a (single) scalar. The shapes of other parameters must be broadcastable with `size`.

Reply:
> Does that make it clear what happens?

Much better! In a rewrite, it would also be nice to include examples of using `rvs`. In most cases, examples are what help me understand the documentation.